src/main/java/com/ecom/shopping/adminkart/controller/ProductController.java

```diff
@@ -0,0 +1,20 @@
+package com.ecom.shopping.adminkart.controller;
+
+import com.ecom.shopping.adminkart.dto.ProductDto;
+import com.ecom.shopping.adminkart.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/product")
+public class ProductController {
```

Comment:
why productcontroller here this PR is only for Category

Reply:
Remove class , not required for this PR

Reply:
please review comment in detail ,didn't get it